#Expyriment 0.9.0 (Python 2.7.15), .xpe-file, coding: UTF-8
#date: Mon Sep 24 2018 09:28:37
#sha1: f626c6
#modules: 
#display: size=(1440, 900), window_mode=False, open_gl=2
#os: ('Darwin', 'Lynns-MacBook-Air.local', '17.2.0', 'Darwin Kernel Version 17.2.0: Fri Sep 29 18:27:05 PDT 2017; root:xnu-4570.20.62~3/RELEASE_X86_64', 'x86_64', 'i386')
Time,Type,Event,Value,Detail,Detail2
11689,design,log,
#design: #exp: First Experiment
#design: block_cnt,block_id,trial_cnt,trial_id
11689,design,logged,
11689,Experiment,started
11709,stdout,received,'height 878'
11709,stdout,received,'width 1440'
11727,stderr,received,'Traceback (most recent call last):'
11728,stderr,received,'  File "/Users/lynn/Desktop/AMT Research assistantship/AMT_results/AMTproject_consistency/testje2.py", line 102, in <module>'
11728,stderr,received,'    s'
11728,stderr,received,"NameError: name 's' is not defined"
